rollup.config.js

```diff
@@ -1,40 +1,77 @@
-import base from './node_modules/@mparticle/web-kit-wrapper/rollup.base';
+// This file is all boilerplate. Do not edit unless you know what you're doing.
```

Comment:
@alexs-mparticle 
changes in this file replicate the latest for our integration example, except for `productionBuilds.rootIife`, i added this to keep an iife at the roote as well as dist/